.github/workflows/run-snapshot-generation.yml

```diff
@@ -0,0 +1,103 @@
+---
+name: Snapshot recording
+
+on:
+  pull_request:
+    types: [ opened, edited , synchronize ]
+
+permissions:
+  checks: write
+  contents: write
+  statuses: write
+  pull-requests: write
+  actions: write
+
+jobs:
+  generate_snapshots:
+    if: ${{contains(github.event.pull_request.title, '[New snapshots]')}}
+    name: Generate snapshots
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+
+      - name: Set up Git LFS
+        run: |
+          git lfs install
+
+      - name: Gradle - Record snapshots with Paparazzi
+        id: testStep
+        run: ./gradlew clean components:recordPaparazziDebug
+
+      - name: Commit and push recorded screenshots
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          branch: ${{ github.event.pull_request.head.ref || github.head_ref || github.ref }}
+          commit_message: "[Paparazzi] Record new snapshots"
+
+      - name: Upload snapshot record report
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: snapshot-recording-failure-report
+          path: /home/runner/work/Loudius/Loudius/components/build/reports/tests/testDebugUnitTest/
+
+      - name: LFS-warning
```

Comment:
question: do we need this if it is already placed in https://github.com/appunite/Loudius/blob/feature/LD-91/golden-tests/.github/workflows/run-snapshot-test.yml#L25?

IF we need this shouldn't be `if: always()`?

Reply:
Yeah, you are right. I am wondering if we need that in the `run-snapshot-test.yml` too because now everything is done on CI. Maybe for now I will remove it only from `run-snapshot-generation.yml`.

Reply:
Oh... Now I understand... because only one of those steps are executed at the same time, this is why you've added LFS-warning to both... this make sense in case of someone added files by them selfs.